miner/block_builder.go

```diff
@@ -91,15 +88,16 @@ type BlockBuilder struct {
 	syncer           syncer
 	started          bool
 	atxsPerBlock     int // number of atxs to select per block
+	txsPerBlock      int // max number of tx to select per block
```

Comment:
there are already 3 parameters passed to the BlocBuilder, I'd consider moving these to a config struct

Reply:
Which ones exactly? The ints?